### PR TITLE
Create two apps with two wsgi instances 

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Create two apps : 
* public
* admin

This is a huge modification as I refactor settings deeply in processing module.

Relate #132

### CODE REVIEW

File: development.ini [hunk 1]
```diff
@@ -3,7 +3,22 @@
 # http://docs.pylonsproject.org/projects/pyramid/en/1.5-branch/narr/environment.html
 ###
 
-[app:main]
+[composite:main]
+use = egg:Paste#urlmap
+/ = consultapp
```
Comment (pgiraud) [inline development.ini]: Can we change `consult` to `public`?
Also I'd remove the `app` suffix in `xxxxxapp`.

Comment (pgiraud) [inline development.ini]: Or maybe `main` is better than `consult`.

**Comment by arnaud-morvan** [comment]
@pgiraud : Travis is happy, this is ready for review now.

File: production.ini [hunk 2]
```diff
@@ -3,7 +3,17 @@
...
-[app:main]
+[app:public]
+use = defaults
+appname = public
+sqlalchemy.url = postgresql://www-data:www-data@localhost:5432/thinkhazard
+
+[app:admin]
+use = defaults
+appname = admin
+sqlalchemy.url = postgresql://www-data:www-data@localhost:5432/thinkhazard_processing
```
Comment (pgiraud) [inline production.ini]: "_" or "-"?
This is not consistent with `development.ini`.

File: tests.ini [hunk 3]
```diff
@@ -1,16 +1,8 @@
...
-jinja2.trim_blocks = true
-jinja2.autoescape = false
-
+[app:public]
+use = config:development.ini#public
 local_settings_path = %(here)s/local.tests.ini
-
-node_modules = %(here)s/node_modules
-
-# path where pdf reports are archived
```
Comment (pgiraud) [inline tests.ini]: Are you sure you want to remove this comment?

Comment (pgiraud) [inline tests.ini]: OK, we're in the `tests.ini`, so we don't really need it.

File: thinkhazard/__init__.py [hunk 4]
```diff
@@ -41,68 +45,77 @@ def main(global_config, **settings):
...
-def load_local_settings(settings):
-    """ Load local/user-specific settings.
-    """
-    local_settings_path = os.environ.get(
-        'LOCAL_SETTINGS_PATH', settings.get('local_settings_path'))
-    if local_settings_path and os.path.exists(local_settings_path):
-        config = ConfigParser.ConfigParser()
-        config.read(local_settings_path)
-        settings.update(config.items('app:main'))
+def add_public_routes(config, prefix):
```
Comment (pgiraud) [inline thinkhazard/__init__.py]: Why not using a keyword arg (ie. prefix='') ?

File: thinkhazard/tests/views/__init__.py [hunk 5]
```diff
@@ -336,6 +336,8 @@ def populate_db():
 
 class BaseTestCase(unittest.TestCase):
 
+    name = 'public'
```
Comment (pgiraud) [inline thinkhazard/tests/views/__init__.py]: Can this be changed to `app_name`?

**Comment by pgiraud** [comment]
@arnaud-morvan I've added one commit to your branch. Can you please give it a quick review?
If OK, then please merge.

**Pull request merged**